samcli/commands/pipeline/init/interactive_init_flow.py

```diff
@@ -107,7 +109,23 @@ def _generate_from_pipeline_template(pipeline_template_dir: Path) -> None:
     pipeline_template: Template = _initialize_pipeline_template(pipeline_template_dir)
     bootstrap_context: Dict = _load_pipeline_bootstrap_context()
     context: Dict = pipeline_template.run_interactive_flows(bootstrap_context)
-    pipeline_template.generate_project(context)
+    with tempfile.TemporaryDirectory() as tempdir:
+        LOG.debug("Generating pipeline files into %s", tempdir)
+        context["outputDir"] = "."  # prevent cookiecutter from generating a sub-folder
```

Comment:
we always set the "outputDir" to "." https://github.com/aws/aws-sam-cli/blob/ae4b8b64fbeb5c87611d64eb0786a702a224fdf7/samcli/lib/cookiecutter/template.py#L149

Reply:
these are two different things. cookiecutter has an `output_dir` and then our template has an `outputDir`.

The former is where the latter is created in.